force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -287,9 +353,14 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
+        Decimal totalCash = 0;
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+        for (Decimal currentCash : moneyInWallet) {
+            if (totalCash > 40) {
+                break;
+            }else totalCash += currentCash;
```

Comment:
Don't forget the opening bracket for thee else. Also its best practice to keep the else and the brackets on their own line:
```
} else {
   totalCash += currentCash;
}
```